editor/src/core/shared/uid-utils.ts

```diff
@@ -20,12 +20,15 @@ import { objectMap } from './object-utils'
 
 export const UtopiaIDPropertyPath = PP.create(['data-uid'])
 
+const GeneratedUIDs: Set<string> = new Set()
```

Comment:
How big will this conceivably get in a session?

Reply:
number of elements times number of times the jsx file is parsed

Reply:
"thanks"

Reply:
Yep, right here we have a memory leak. If we're going to use this we need to clear the `Set` on each parse or at some other sensible time.

Reply:
too bad Set doesn't support a "max size" option, in a FIFO kind of way

Reply:
I talked to Sean and created a follow-up ticket: https://github.com/concrete-utopia/utopia/issues/668

in the meantime I'll merge this as-is to unblock us from testing the editor.

Reply:
Sounds good to me :+1: